activation.md

```diff
@@ -1,4 +1,28 @@
 ---
 description: >-
   Descibes the licencing of SysKit Sense and how to activate the solution.
----
\ No newline at end of file
+---
+
+SysKit Sense will not work without a valid license and any interaction with the website will be blocked by the license screen.  
+You must be a member of the SysKit Sense Admin role in Azure Active Directory to license the product. Learn more about the application roles [here](../security/application-roles.md).
+
+Once a new license key has been provided the license will be immediately applied, no further actions are required.
+The license is tied to the Azure Active Directory tenant where SysKit Sense is deployed.  
+
+An evaluation license key should have been provided during the initial SysKit Sense deployment. The Evaluation license is valid for 30 days.
+
+
+## Editions
+There are two editions of SysKit Sense.
+
+### Evaluation
+Contains all of the Enterprise features but limited to 30 days. If you require a longer time to evaluate the product please contact us for an extension.

```

Comment:
Maybe we should put some link here for the Contact Us?

Reply:
yeah, i thought about it, do we have some kind of an endpoint in mind? Cant seem to find anything similar on the SSM web. only references to https://www.syskit.com/company/contact-us

Reply:
we could use the some app link here so we can redirect it to the right form. same app link that we are going to use in the application on licensing screen?

Reply:
renamed the file. check the new one for the change.